### ISSUE TITLE
[WINE] Crash when a world disconnects you when connected via TLS

### ISSUE THREAD

**Issue by lisdude** [BeipDev/BeipMU#12]
This one is pretty easy to reproduce. Using WINE, connect to a world's TLS port and then get the server to disconnect you (e.g. @quit in MOO). 

[backtrace.txt](https://github.com/BeipDev/BeipMU/files/2070085/backtrace.txt)

**Comment by BeipDev** [comment]
I installed Kubuntu 18.04 and then installed Wine then tried a bunch of SSL places I had but I couldn't get it to crash. Was there a particular place that did it?

I also noticed that when typing 'QUIT' the mu* acts like it's disconnecting but Wine doesn't tell BeipMU the socket is closed. I tried a few approaches but I couldn't get it to report that the socket had closed. This used to work in an older version of Wine.
